example/README.md

```diff
@@ -2,12 +2,15 @@ Example GraphiQL Install
 ========================
 
 This example uses the version of GraphiQL found in the parent directory, rather
-than depending on npm, so that it is easier to test new changes. In order to use
-the compiled version of GraphiQL, first run build in the parent directory before
-installing and starting the example.
+than depending on npm, so that it is easier to test new changes. In order to use this
+example as a basis for development, you must have at least Node 6 installed.
 
-1. Run `npm install && npm run build` in the parent directory
-2. Navigate to this directory (example) in Terminal
+Instructions:
```

Comment:
This is still quite a few instructions - maybe we should just merge these packages together?